packages/tools/guiEditor/src/controlTypes.ts

```diff
@@ -91,3 +91,80 @@ export const ControlTypes = [
         icon: colorPickerIcon,
     },
 ];
+
+export const ControlTypesFHalf = [
+    {
+        className: "Rectangle",
+        icon: rectangleIcon,
+    },
+    {
+        className: "Ellipse",
+        icon: ellipseIcon,
+    },
+    {
+        className: "StackPanel",
+        icon: stackPanelIcon,
+    },
+    {
+        className: "Grid",
+        icon: gridIcon,
+    },
+    {
+        className: "ScrollViewer",
+        icon: scrollbarIcon,
+    },
+    {
+        className: "Line",
+        icon: lineIcon,
+    },
+    {
+        className: "TextBlock",
+        icon: textBoxIcon,
+    },
+    {
+        className: "InputText",
+        icon: inputFieldIcon,
+    },
+    {
+        className: "InputPassword",
+        icon: passwordFieldIcon,
+    },
+];
+export const ControlTypesSHalf = [

```

Comment:
What is the difference between SHalf and FHalf? Is the difference needed? can both be in a single array?